common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2433,20 +2433,24 @@ StScrollBar {
 //
 .user-icon {
   background-size: contain;
-  color: $fg_color;
+  color: $osd_fg_color;
   border-radius: 99px;
+
+  .login-dialog-user-list-item:selected & { color: $selected_fg_color; }
+
   &:hover {
-    color: transparentize($fg_color, 0.3);
+    color: if($variant != 'lighter', lighten($osd_fg_color, 25%), darken($osd_fg_color, 12%));
   }
```

Comment:
I'm not sure if the special styling fo `:hover` is necessary here? If it is, I think we should retain the unchanged `$osd_fg_color` here as well.

Reply:
The styling for `:hover` has no impact that I can see, just left in for consistency with upstream - but agree on using `$osd_fg_color`